src/ipolates.f90

```diff
@@ -95,293 +104,293 @@ SUBROUTINE IPOLATES_grib1(IP,IPOPT,KGDSI,KGDSO,MI,MO,KM,IBI,LI,GI, &
     call ipolates_grid(ip, ipopt, grid_in, grid_out, mi, mo, km, ibi, li, gi, no, rlat, rlon, ibo, lo, go, iret)
 
   END SUBROUTINE IPOLATES_GRIB1
-  
 
+
+  !$$$  SUBPROGRAM DOCUMENTATION BLOCK
+  !
```

Comment:
This and all other such documentation blocks must be converted to doxygen. Lower-case it as well.